packages/doctype-caip10-link-handler/src/__tests__/caip10-link-doctype-handler.test.ts

```diff
@@ -0,0 +1,264 @@
+import { Caip10LinkDoctypeHandler } from '../caip10-link-doctype-handler'
```

Comment:
I'm just assuming that this file is identical to the old version and this is just a file rename.  If anything actually changed in this file let me know and I can review that piece

Reply:
Same.